app/models/storages.py

```diff
@@ -0,0 +1,18 @@
+from db import Base
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+
+
+class Storage(Base):
+    __tablename__ = 'storages'
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    product_amount = sa.Column(sa.String(45), nullable=False)
+    min_amount = sa.Column(sa.String(100), nullable=False)
+    reorder_amount = sa.Column(sa.Integer, nullable=False)
+    shop_id = sa.Column(sa.Integer, sa.ForeignKey('shops.id'))
```

Comment:
Should be primary key

Reply:
VAD är det som skall vara PK?

Reply:
Nu är id satt till FK så att de två foreign key skapar en unik PK.